src/ceph-daemon/ceph-daemon

```diff
@@ -559,7 +564,7 @@ def extract_uid_gid():
     return (int(uid), int(gid))
 
 def deploy_daemon(fsid, daemon_type, daemon_id, c, uid, gid,
-                  config, keyring):
+                  config=None, keyring=None):
```

Comment:
This will make mypy unhappy :(

```
mypy installed: mypy==0.740,mypy-extensions==0.4.3,typed-ast==1.4.0,typing-extensions==3.7.4.1
mypy run-test-pre: PYTHONHASHSEED='495726843'
mypy run-test: commands[0] | mypy ceph-daemon
ceph-daemon:575: error: Argument 1 to "write" of "IO" has incompatible type "Optional[str]"; expected "str"
ceph-daemon:582: error: Argument 1 to "write" of "IO" has incompatible type "Optional[str]"; expected "str"
ceph-daemon:610: error: Argument 1 to "write" of "IO" has incompatible type "Optional[str]"; expected "str"
Found 3 errors in 1 file (checked 1 source file)
```